corehq/apps/app_manager/suite_xml/features/case_tiles.py

```diff
@@ -181,14 +182,14 @@ def _get_column_context(self, column):
             "prefix": escape(
                 column.header.get(default_lang, "")
             ),
-            "format": column.format
+            "format": column.format,
+            "variables": '',
+            "endpoint_action": '',
```

Comment:
was `endpoint_action` just missing?

Reply:
no, both `variables` and `endpoint_action` were set as blank strings later down the function so I just grouped them into the initial dictionary creation